Session/Settings/Views/VersionFooterView.swift

```diff
@@ -33,12 +33,19 @@ class VersionFooterView: UIView {
         result.lineBreakMode = .byCharWrapping
         result.numberOfLines = 0
         
-        if
-            let version: String = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String,
-            let buildNumber: String = Bundle.main.infoDictionary?["CFBundleVersion"] as? String
-        {
-            result.text = "Version \(version) (\(buildNumber))"
-        }
+        let infoDict = Bundle.main.infoDictionary
+        let version: String = ((infoDict?["CFBundleShortVersionString"] as? String) ?? "0.0.0")   // stringlint:disable
+        let buildNumber: String? = (infoDict?["CFBundleShortVersionString"] as? String)           // stringlint:disable
+        let commitInfo: String? = (infoDict?["GitCommitHash"] as? String)                         // stringlint:disable
```

Comment:
Do we want to always show the git commit hash info? Or just for test builds? It doesn't make sense to me to show this info to all users.

Reply:
Yep - both Android and Desktop are doing this so we are updating iOS to be consistent (it doesn't hurt to show it and I guess also means anyone that knows about the Github project can also associate the build with a specific commit)

It also means that internal test builds on TestFlight can be distinguished via the commit hash so forgetting to manually updating the build number between test builds is less likely to cause confusion 🤞